### PR TITLE
Change CI must-gather-network to use oc adm

### PR DESCRIPTION
**Pull request by pecameron**
SDN-988 Change CI must-gather-network to use oc adm

Signed-off-by: Phil Cameron <pcameron@redhat.com>

### CODE REVIEW

**Comment by pecameron** [comment]
@wking @JacobTanenbaum PTAL
This PR is for data collection during post. The failures are not relevant to this working. Look through the artifacts to see the collected files.

File: ci-operator/step-registry/gather-network/gather-network-commands.sh [hunk 1]
```diff
@@ -1,99 +1,16 @@
...
-export PATH=$PATH:/tmp/shared
 
 if test ! -f "${KUBECONFIG}"
 then
 	echo "No kubeconfig, so no point in gathering extra artifacts."
 	exit 0
 fi
 
-echo "Gathering network ovn artifacts ..."
+echo "Gathering network ovn artifacts ... "
```
Comment (JacobTanenbaum) [inline ci-operator/step-registry/gather-network/gather-network-commands.sh:10]: nit: extra space before ending double quote

Comment (pecameron) [reply ci-operator/step-registry/gather-network/gather-network-commands.sh]: Changed.

**Pull request merged**